.github/workflows/executor-cd.yml

```diff
@@ -0,0 +1,33 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+#  pull_request:
+# uncoment the line above to test CD push in a PR
+
+jobs:
+  push-executors:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get previous tag
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: ""
+      - name: push to Hubble
+        id: push_exec
+        uses: jina-ai/action-hubble-push@v2
+        with:
+          git_tag: ${{ steps.previoustag.outputs.tag }}
+        env:
+          exec_uuid: ${{ secrets.UUID }}
+          exec_secret: ${{ secrets.SECRET }}
```

Comment:
I don't think this is how versioning should work. If we want github tag = version, then it only makes sense to version when we assign a tag.